src/green_recognition/CMakeLists.txt

```diff
@@ -0,0 +1,84 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(green_recognition)
+
```

Comment:
Just a note: I'm OK with keeping this and line detection separate for now, but we should likely eventually just put them in the same project.